kuma/core/email_utils.py

```diff
@@ -35,9 +33,9 @@ def uselocale(locale):
     run uselocale---the locale will already be set correctly.
```

Comment:
Now that we use Django's i18n tools we can just remove this function and use `django.utils.translation.override`: https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#using-translations-outside-views-and-templates